src/randoop/JunitFileWriter.java

```diff
@@ -36,10 +35,12 @@
 
   private Map<String, List<List<ExecutableSequence>>> createdSequencesAndClasses = new LinkedHashMap<String, List<List<ExecutableSequence>>>();
 
+  private Map<String, Integer> methodCountMap = new LinkedHashMap<String,Integer>();
+
   public JunitFileWriter(String junitDirName, String packageName, String junitDriverClassName, int testsPerFile) {
```

Comment:
Given that you changed the field junitDriverClassName to masterTestClassName, shouldn't the formal parameter name be changed too?